src/load.c

```diff
@@ -362,6 +369,136 @@ int xmp_test_module(char *path, struct xmp_test_info *info)
 	return ret;
 }
 
+int xmp_test_module_from_memory(void *mem, long size, struct xmp_test_info *info)
+{
+	HIO_HANDLE *h;
+	char buf[XMP_NAME_SIZE];
+	int i;
+	int ret = -XMP_ERROR_FORMAT;
+
+	/* Use size < 0 for unknown/undetermined size */
+	if (size == 0)
+		size--;
+
+	if ((h = hio_open_mem(mem, size)) == NULL)
+		return -XMP_ERROR_SYSTEM;
+
+#ifndef LIBXMP_CORE_PLAYER
+	/* get size */
+	if (hio_size(h) < 256) {	/* set minimum valid module size */
```

Comment:
I don't know why this is needed? AFAIK, `xmp_load_module_from_memory` doesn't have it.